azurerm/internal/services/network/vpn_gateway_resource.go

```diff
@@ -78,6 +79,45 @@ func resourceArmVPNGateway() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+
+						"bgp_peering_addresses": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"ip_configuration_id": {
```

Comment:
Should this be called something else? ip configuration id implied an azure resource, when its a text string? 

what does this value represent and is there a more descriptive/accurate name?